pkg/clusteragent/languagedetection/README.md

```diff
@@ -0,0 +1,14 @@
+# Language Detection Patcher
```

Comment:
This contains the same description as the `doc.go` file so I think one should be removed.